components/menu/_menu.scss

```diff
@@ -1,71 +1,54 @@
 .sdc-menu-list {
     position: fixed;
-    z-index: 100;
 
-    border-style: solid;
-    border-width: 1px;
-    border-color: $light-gray;
+    @include box-sizing;
+    @include box-shadow(0 2px 4px 0 rgba($black, 0.3));
+    border: 1px solid $light-gray;
     border-top: solid 3px $blue;
-    box-sizing: border-box;
-    background-color: $white;
-    width: 150px;
     border-radius: 2px;
-    @include box-shadow(0px 2px 4px 0px rgba($black, 0.3));
+    background: $white;
+    width: 150px;
 
     .sdc-menu-item {
-        font-size: 14px;
-        line-height: 29px;
+        @extend .body-1;
+        height: 29px;
         color: $dark-gray;
-        g, path{
-            fill: $dark-gray;
-        }
         padding: 0 12px;
+        display: flex;
+        align-items: center;
+
+        svg {
+            margin-right: 5px;
+            g, path {
+                fill: $dark-gray;
+            }
+        }
 
         &:hover {
-            cursor: pointer;
-            background-color: $tlv-hover;
-            color: $hover-blue;
-            g, path{
-                fill: $hover-blue;
+            &:not(.disabled) {
+                cursor: pointer;
+                &:not(.selected) {
+                    background: $background-gray;
+                    color: $text-black;
+                    g, path {
+                        fill: $text-black;
+                    }
+                }
             }
         }
 
         &.disabled {
-            cursor: default;
             color: $gray;
-            g, path{
+            g, path {
                 fill: $light-gray;
             }
-
-            &:hover {
-                cursor: default;
-                background-color: inherit;
-            }
         }
 
         &.selected {
-            background-color: $background-gray;
-            color: $text-black;
-            g, path{
-                fill: $text-black;
-            }
-        }
-
-        &.line {
```

Comment:
What's the reason to remove `&.line` and put `.separator` outside `.sdc-menu-item` ?
You could just rename `.line` into `.separator`.

Reply:
I spoke with Israel, Einav, and Fridman about this. We decided separator is a clearer name. It is "outside" because the separator is not a menu item. It separates the items.

Reply:
OK, that makes sense.